### ISSUE TITLE
Target size of buttons available under Kiota openapi explorer is less than required size 24x24pixel: A11y_VS Code_Target Size

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#44541]
--- GitHub Tags
-A11yTCS; -A11ySev2; -WCAG2.5.8; -DesktopApp; -Visual Studio Code Client; -BM-VisualStudioCodeClient-Win32-Jan2024; -FTP; -Benchmark; -Win32; -A11yMAS; -A11yWCAG2.2;

--- Environment Details:
Application Name: VS Code Extension for API Center
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)

**Pre-requisites:**
Open system settings.
Navigate to Bluetooth & devices->Mouse->Additional mouse settings.
Tab to Pointers tab.
Tab to browse and hit enter.
Select the 24*24.cur file(please find in attachment).
The mouse pointer will change to 24x24 pixel box.
You can move the pointer and verify the size of any control.
--- Repro Steps:
Create a API center in azure portal.
Open Visual studio Code.
3.Tab till API center extension in theleft side panel and press enter.
Tab till definition under the api name which you have created and press enter.
The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
Go to "Generate API Client" using the down arrow and press enter.
Kiota openapi explorer window will open.
Observe that Target size of buttons available under Kiota openapi explorer is less than required size 24x24pixel.
--- Actual Results:
Target size of buttons available under Kiota openapi explorer is less than required size 24x24pixel

--- Expected Results:
Target size of buttons available under Kiota openapi explorer should be minimum 24x24pixel. If the controls have spaced in-between, then those spaces should not intersect each other and target size with spacing should be 24x24 pixel for each one.

--- User Impact:
Users with dexterity limitations and those who have difficulty with fine motor movement find it difficult to accurately activate small targets when there are other targets that are too close. Providing sufficient size, or sufficient spacing between targets, will reduce the likelihood of accidentally activating the wrong control.

--- Attachment:
